ehr/api-src/org/labkey/api/ehr/table/VetReviewDisplayColumn.java

```diff
@@ -44,7 +47,13 @@ public void renderGridCellContents(RenderContext ctx, Writer out) throws IOExcep
                     text = text.replaceAll("\\*\\*", "<span style=\"background-color: yellow;\">\\*\\*</span>");
                 }
 
-                out.write("<a style=\"max-width: 500px;\" onclick=\"EHR.panel.ClinicalManagementPanel.replaceSoap({objectid: " + PageFlowUtil.jsString(StringUtils.trimToNull(tokens[2])) + ", scope: this, callback: function(){EHR.panel.ClinicalManagementPanel.updateVetColumn(this, arguments[0], arguments[1]);}});\">");
+                out.write("<a style=\"max-width: 500px;\" class=\"labkey-text-link vrdc-row\" data-objectid=" + PageFlowUtil.jsString(StringUtils.trimToNull(tokens[2])) + ">");
```

Comment:
jsString won't handle all possible values
```suggestion
                out.write("<a style=\"max-width: 500px;\" class=\"labkey-text-link vrdc-row\" data-objectid=\"" + PageFlowUtil.filter(StringUtils.trimToNull(tokens[2])) + "\">");
```